megapowerani.com.template.json

```diff
@@ -0,0 +1,70 @@
+{
+    "providerId": "megapowerani.com",
+    "providerName": "Freshsales Test",
+    "serviceId": "template",
+    "serviceName": "Domain verification",
+    "version": 1,
+    "logoUrl": "https://avatars.slack-edge.com/2021-08-02/2349087959585_442632783598439baea4_512.png",
+    "description": "Freshsales Domain Verification Test",
+    "variableDescription": "This is  atest for Freshsales domain verification using Domain Connect",
```

Comment:
there are no variables, so `variableDescription` seems to be confusing. Likely this should be in `description` instead

Reply:
I will leave variableDescription as empty